daemon/cmd/kube_proxy_replacement.go

```diff
@@ -428,7 +428,7 @@ func handleNativeDevices(strict bool) {
 		option.Config.DirectRoutingDevice == ""
 	if detectNodePortDevs || detectDirectRoutingDev {
 		if err := detectDevices(detectNodePortDevs, detectDirectRoutingDev); err != nil {
-			msg := "Unable to detect devices for BPF NodePort."
+			msg := "Unable to detect devices to attach bpf_host."
```

Comment:
I'm not sure that will speak to users. Maybe we should instead explicitly list features which require native devices?

Reply:
Yeah, makes sense. Let me add it.